src/routes/Room.js

```
@@ -0,0 +1,75 @@
+import { IntlProvider, Text } from 'preact-i18n';
+import { PureComponent }      from 'react';
+import { connect }            from 'react-redux';
+import { bindActionCreators } from 'redux';
+import Promise                from 'bluebird';
+import { ProgressBar }        from 'react-toolbox/lib/progress_bar';
+import FeaturesDetails           from '~/containers/room/FeaturesDetails';
+import * as actions           from '~/actions';
+
+
+class Room extends PureComponent {
+  componentWillMount() {
+    const { roomId, actions } = this.props;
+
+    Promise.resolve()
+      .then(() => actions.getRoom(roomId))
+      .then(({ response }) =>  actions.listFeatures(roomId, response.included[0].id))
+      .catch((e) => console.log(e));
```

You either make something useful of that error, or you use `rethrow: false` in the action. You don't silence the error to the console.